ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -0,0 +1,924 @@
+  local charts_dir="${SCRATCH}/charts"
+
+  # Option 1: Extract from pre-built container image (preferred, works with restrict_network_access: true)
+  # ci-operator provides built images via IMAGE_FORMAT and IMAGE_TRUSTEE_CHARTS env vars
+  if [[ -n "${IMAGE_TRUSTEE_CHARTS:-}" ]]; then
+    local charts_image="${IMAGE_TRUSTEE_CHARTS}"
+    echo ">>> Extracting trustee charts from pre-built image"
+    echo ">>> Image: ${charts_image}"
+
+    # Extract charts from the image
+    mkdir -p "${charts_dir}"
+    if oc image extract "${charts_image}" --path /charts/:${charts_dir}/ 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)